src/app/component/Field/Field.component.js

```diff
@@ -332,7 +365,14 @@ Field.propTypes = {
         PropTypes.func,
         PropTypes.shape({ current: PropTypes.instanceOf(Element) })
     ]),
-    isAutocompleteAllowed: PropTypes.bool
+    isAutocompleteAllowed: PropTypes.bool,
+    countries: PropTypes.arrayOf(
```

Comment:
Why abstract select options are called countries?

Reply:
Changed name to countryList